app/src/main/java/org/mozilla/fenix/home/pocket/PocketCategoriesViewHolder.kt

```diff
@@ -121,21 +119,18 @@ private fun PocketTopics(
         PocketStoriesCategories(
             categories = categories,
             selections = categoriesSelections,
-            selectedTextColor = selectedTextColor,
-            unselectedTextColor = unselectedTextColor,
-            selectedBackgroundColor = selectedBackgroundColor,
-            unselectedBackgroundColor = unselectedBackgroundColor,
+            modifier = Modifier.fillMaxWidth(),
+            categoryColors = categoryColors,
             onCategoryClick = onCategoryClick,
-            modifier = Modifier
-                .fillMaxWidth(),
         )
     }
 }
 
 @Composable
-@Preview
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Preview(uiMode = Configuration.UI_MODE_NIGHT_NO)
 private fun PocketCategoriesViewHolderPreview() {
-    FirefoxTheme(theme = Theme.getTheme()) {
+    FirefoxTheme {
```

Comment:
I didn't verify this, but assuming we already have a default value for `theme`. Let's file a followup to clean all of this up (eg, removing all the `(theme = Theme.getTheme())`) if there's still any leftover in the codebase.

Reply:
We have a default, and yeah, a refactor a few months back added all of the `theme = Theme.getTheme()` calls.

Reply:
Opened: 
- https://github.com/mozilla-mobile/fenix/issues/27854